API/models-json/Open-Ethics-Data-Passport.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
@abhi12ravi the header of our every schema module/json file should contain required metadata

1) contain `$id` (we'll later upload it to openethics.ai for everyone to see) and will submit to the schemastore.org
2) contain `  "$schema": "https://json-schema.org/draft/2020-12/schema"`

as defined in the metaschema specs https://json-schema.org/draft/2020-12/release-notes